lookbook/docs/patterns/18-enterprise-components.md.erb

```diff
@@ -0,0 +1,81 @@
+OpenProject provides multiple ways to render upsale components. This document describes the different components and how to use them.
```

Comment:
I think this should be moved to the components folder `lookbook/docs/components`